detection_rules/etc/version.lock.json

```diff
@@ -9382,16 +9796,25 @@
   "ff4599cb-409f-4910-a239-52e4e6f532ff": {
     "min_stack_version": "8.7",
     "rule_name": "LSASS Process Access via Windows API",
-    "sha256": "3ebb73fb1bc78e99a7321c9da744e2462cb56b7b8b3a372342993176f40608c2",
+    "sha256": "45523e08c1b08b3aeb6e316fbfd73c257194c643b9c2d30533a4c05de668ca18",
     "type": "eql",
-    "version": 6
+    "version": 7
   },
   "ff4dd44a-0ac6-44c4-8609-3f81bc820f02": {
-    "min_stack_version": "8.3",
+    "min_stack_version": "8.8",
+    "previous": {
+      "8.3": {
+        "max_allowable_version": 205,
+        "rule_name": "Microsoft 365 Exchange Transport Rule Creation",
+        "sha256": "e247dbb68f81f5c55155bea1dd2a757717bdc740b8259a933165e5a612d3cdb7",
+        "type": "query",
+        "version": 106
+      }
+    },
     "rule_name": "Microsoft 365 Exchange Transport Rule Creation",
     "sha256": "24df1fab9f47005a3dcf144bdd7993c237e1da4de8b6ed8ee44d4513417e0f88",
     "type": "query",
-    "version": 105
```

Comment:
Does this bump make sense?

Reply:
We have had a similar bump in the past as well. 
https://github.com/elastic/detection-rules/pull/3567/files#diff-d539b5f6ddb25f5e1ed43553241cee8ba802cd4417097d8fafcb2f90b557b935L8197
Looks consistent with that.

Reply:
As confirmed by @terrancedejesus now they have 2 distinct hashes where the current will always bump +1 and the forked versions will never change.

Reply:
Then that would indicate that there is something wrong given the bump from this comment right? https://github.com/elastic/detection-rules/pull/3567#discussion_r1548300981

Reply:
No even that rule had a minstack version change https://github.com/elastic/detection-rules/pull/3567#discussion_r1548306849. this was not related to the o365 integration changes we pushed. it was from the PR https://github.com/elastic/detection-rules/pull/3505